lib/index.js

```diff
@@ -289,7 +289,13 @@ async function _checkStatuses({
     })));
   const verified = results.every(
     ({verified = false} = {}) => verified === true);
-  return {verified};
+  const revocation = results.find(r => r.purpose == 'revocation')
+  const suspension = results.find(r => r.purpose == 'suspension')
+  return {
+          verified, 
+          ...(revocation && { revocation }),
+          ...(suspension && { suspension })
```

Comment:
Since the plan is still (I believe) to enable other custom purposes with status lists ... we should nest the results to avoid mixing / conflation. So perhaps something like this?:

```
{
  verified,
  purposes: {
    revocation: {verified, ...},
    suspension: {verified, ...}
  }
}
```

Though maybe we just want to return: `{verified, results}`.

Reply:
I think results is the better choice here. But in the end it is up to you. Results is an array then?